packages/strapi-plugin-content-manager/admin/src/components/TableDelete/index.js

```diff
@@ -10,8 +10,9 @@ import { FormattedMessage } from 'react-intl';
 
 import styles from './styles.scss';
 
-function TableDelete({ colspan, number, onToggleDeleteAll }) {
+function TableDelete({ colspan, number, total, onToggleDeleteAll }) {
   const suffix = number > 1 ? 'plural' : 'singular';
+  const deleteMessageId = number === total ? 'delete' : 'deleteSelected';
```

Comment:
Do we really consider it is a "delete all" if we only delete all visible items but we have more items in an other page.

e.g: select all 10 of first page among 100 pages does it really count as a "delete all" action ? IMO I don't think so

Reply:
what do you think @soupette

Reply:
Well if you understand it as delete all item from this page if do believe that it is correct.

Reply:
Is 'Delete all (on page)' a better alternative? Or is that too verbose?

Reply:
too verbose I think. You can keep it like that If @soupette prefers

Reply:
Yeah it's a bit too verbose.